meinberlin/templates/a4dashboard/base_dashboard_project.html

```diff
@@ -187,8 +187,3 @@ <h2 class="modal-title u-first-heading" id="delete-modal-label">{% trans "Are yo
         </div>
     </div>
 {% endblock %}
-
-
-{% block extra_js %}
-    <script src="{% static 'js/unload_warning.js' %}"></script>
```

Comment:
While I do agree with this change here, it's a bit odd to load one big monolithic js file everywhere. But for our current webpack setup it's probably the best approach :/

Reply:
Some for the ajax modal btw.

Reply:
I agree, but we should come up with a generell rule (maybe we could have webpack bundles for each app) here, and the unload_warning actually makes sense for all forms I would say the ajax_modal not so much.